scripts/fast-reboot-dump.py

```diff
@@ -11,39 +11,46 @@
 import argparse
 import syslog
 import traceback
+import ipaddress
 
 
 ARP_CHUNK = binascii.unhexlify('08060001080006040001') # defines a part of the packet for ARP Request
 ARP_PAD = binascii.unhexlify('00' * 18)
 
-def generate_arp_entries(filename, all_available_macs):
+def generate_neighbor_entries(filename, all_available_macs):
     db = swsssdk.SonicV2Connector(host='127.0.0.1')
     db.connect(db.APPL_DB, False)   # Make one attempt only
 
     arp_output = []
-    arp_entries = []
+    neighbor_entries = []
     keys = db.keys(db.APPL_DB, 'NEIGH_TABLE:*')
     keys = [] if keys is None else keys
     for key in keys:
         vlan_name = key.split(':')[1]
-        ip_addr = key.split(':')[2]
         entry = db.get_all(db.APPL_DB, key)
-        if (vlan_name, entry['neigh'].lower()) not in all_available_macs:
+        mac = entry['neigh'].lower()
+        if (vlan_name, mac) not in all_available_macs:
             # FIXME: print me to log
             continue
         obj = {
           key: entry,
           'OP': 'SET'
         }
-        arp_entries.append((vlan_name, entry['neigh'].lower(), ip_addr))
         arp_output.append(obj)
 
+        ip_addr = key.split(':')[2]
+        if ipaddress.ip_interface(ip_addr).ip.version != 4:
```

Comment:
Yes. This is a necessary modification in 201911 to handle IPv6 entries.
But there is some problem using the `ipaddress.ip_interface(ip_addr).ip.version` to parse it.
From my opinion, I will consider using some simple checking like
`if '.' not in ip_addr:`
If no '.' inside the ip_addr, it is a IPv6 address.

Reply:
Since we identified there is problem with script, why is it taken to 201911? @qiluo-msft, @tahmed-dev, @abdosi  . This is breaking the 201911 fastboot.

Reply:
@prsunny Do you have a Github issue? Are you suggesting revert?

Reply:
I will suggest replace the line ` if ipaddress.ip_interface(ip_addr).ip.version != 4:` with this line `if '.' not in ip_addr:` to handle the IPv6 entries. And also fix the problem here.
Simple is powerful.

Reply:
Checking with @bingwang-ms to see if this can be cherry-picked - https://github.com/Azure/sonic-utilities/pull/1164